packages/admin-ui/ui/src/components/organisms/batch-jobs-activity-list/index.tsx

```diff
@@ -86,12 +86,8 @@ const BatchJobActivityCard = (props: { batchJob: BatchJob }) => {
     to: batchJob.created_at,
   })
 
-  const operation = {
-    "product-import": BatchJobOperation.Import,
-    "price-list-import": BatchJobOperation.Import,
-    "product-export": BatchJobOperation.Export,
-    "order-export": BatchJobOperation.Export,
-  }[batchJob.type]
+  let operation = batchJob.type.split("-").pop()
```

Comment:
@pepijn-vanvlaanderen can you add a small description of the formatting for future reference? Would make it easier for people reading the code to quickly know what to do